project-base/templates/Front/Content/Product/detail.html.twig

```diff
@@ -146,7 +146,7 @@
                     </div>
                 </div>
 
-                {% if product.isMainVariant %}
+                {% if product.isMainVariant and not product.calculatedSellingDenied %}
```

Comment:
But last thing - I see chage in project-base but I don't see upgrade notes.

Reply:
Thx, upgrade notes added.